tencentcloud/provider.go

```diff
@@ -658,6 +658,7 @@ func Provider() terraform.ResourceProvider {
 		DataSourcesMap: map[string]*schema.Resource{
 			"tencentcloud_availability_regions":                     dataSourceTencentCloudAvailabilityRegions(),
 			"tencentcloud_availability_zones":                       dataSourceTencentCloudAvailabilityZones(),
+			"tencentcloud_availability_zones_by_product":            dataSourceTencentCloudAvailabilityZonesByProduct(),
```

Comment:
you also need add `tencentcloud_availability_zones_by_product` above to genernate doc.
`go build;./gendoc`